src/goto-checker/solver_factory.h

```diff
@@ -50,19 +50,18 @@ class cbmc_solverst
     {
     }
 
-    explicit solvert(std::unique_ptr<prop_convt> p):prop_conv_ptr(std::move(p))
+    explicit solvert(std::unique_ptr<prop_convt> p)
```

Comment:
Not directly related to this PR but I find it a bit strange that the product of the factory is defined in the factory itself. And it seems also this should have been factory functions rather than a factory class as most fields are constant and I guess it's only used to produce one solver.

Reply:
Noted here: https://github.com/diffblue/cbmc/issues/3574